### ISSUE TITLE
Sandbox workflow wording

### ISSUE THREAD

**Issue by odinuv** [keboola/kbc-ui#2003]
is highly confusing, first:
![image](https://user-images.githubusercontent.com/4319320/45268359-508a0600-b47b-11e8-8297-e90356cab25b.png)

clicking new sandbox can create a new sandbox or return existing one. then i see this:

![image](https://user-images.githubusercontent.com/4319320/45268370-744d4c00-b47b-11e8-9dd8-e90ec5c7e854.png)

so it's creating a sandbox before i hit the "Create sandbox" button (even before it is enabled) and then i get

![image](https://user-images.githubusercontent.com/4319320/45268379-93e47480-b47b-11e8-9588-175faaa337bc.png)

so i can hit the create button when the sandbox is obviously created

proposal:
"New Sandbox" -> "Sandbox"
"Creating Sandbox..." -> "Getting Sandbox Credentials..."
"Create Sandbox" -> "Load data"

**Comment by janmichek** [comment]
Diky za reportnuti a navrh. Mrknu se na to

**Comment by janmichek** [comment]
Proklikal jsem si loadovani sandboxu.
Ui a poradi buttonu neni kdovijak navodny k linearni user flow, to ted nebudeme resit.

Mam jenom jeden navrh na zmenu.

"New Sandbox" -> "Load to Sandbox" - rad bych tam videl nejaky sloveso, protoze se jedna o akci.
Co mysilis? @odinuv

**Comment by odinuv** [comment]
Kdyz vytvoris sandbox u transformace tak existuje furt dal, stisknutim nekterych z tech tlacitek ho cely smazes (a prepises) Kvizova otazka: kterym? :)

**Comment by janmichek** [comment]
Vyzkousim napovedu pritele na telefonu

**Comment by odinuv** [comment]
tlacitkem Create Sandbox :) (to je logicky, ne?) :)

ale rozhodne to neni tim tlacitkem New Sandbox - to prave jen zobrazi credentials, cili pokud tam ma byt nejaka akce, tak "Get Sandbox (Credentials)" - nic vic to nedela

**Comment by janmichek** [comment]
Zkusil jsem pripravit tu zmenu.
Vetsinou mi to prijde navodnejsi, ale tady moc ne.  V Sandbox listu - otevreny pres bucket  by zase 'New Sandbox' sedelo dobre. Ty ikony taky moc nesedi.

![image](https://user-images.githubusercontent.com/15363559/45548555-f5626580-b824-11e8-8b7b-6b8128525fac.png)

Cela to workflow Sandboxu mi pripada mizerna (ruzny backendy, ruzny ovladani)

**Comment by odinuv** [comment]
no ruzny backendy a ruzny ovladani to mit bude jeste dlouho, ale i u plain sandboxu plati stejna logika - tlacitkem "New Sandbox" se sandbox nevytvori, ale pouze nactou credentials, sandbox se vytvori az stistknutim "+ Sandbox" - cili ta akce je opet uvnitr modalu ne na odkazu v pravym menu

**Comment by janmichek** [comment]
Moc nevim jak to efektivne vyresit. Na screenu jsem ciste zaimplementoval tvuj navrh. Myslim ze se nam nedari najit jednotny nazvy, kvuli nejednote ovaldani.

**Comment by odinuv** [comment]
no, ale to ovladani je prave celkem jednotny (tedy v tom kdy se co stane), kde ti to nesedi?

**Comment by janmichek** [comment]
(uvazuju novy nazvoslovi)

Snowflake
+ Sandbox -> Getting Sandbox Credentials -> Load Data -> modal -> Run -> Connect

Python
+ Sandbox -> modal -> Load Data ->Getting Sandbox Credentials -> Connect

**Comment by odinuv** [comment]
tak nejdriv soucasny stav:
docker transformace:
New Sandbox -> {modal} -> Looking for existing sandbox (coz se opravdu jen diva jestli existuje) -> Create Sandbox -> Creating sandbox and loading data... {vytvari sandbox credentials a nahrava data} -> Connect {otevre novy okno}

sql transformace:
New Sandbox -> {modal} -> Creating Sandbox (coz se diva jestli existuji sandbox credentils a pokud ne, tak vytvori novy) -> Create Sandbox -> Loading Data into sandbox (nahrava (a pripadne prepisuje) data) -> Connect {otevre novy okno}

plain sandbox:
docker transformace:
Sandbox -> + new Sandbox -> Modal s vyberem tabulek -> Creating Sandbox (vytvari sandbox credentials a nahrava data) -> Connect (otevre novy okno)

sql transformace:
Sandbox -> + new Sandbox -> Creating Sandbox (vytvari sandbox credentials) -> Connect (otevre novy okno)

takze u toho plain sandbox mi to prijde v pohode, u transformacniho sandboxu je to "New Sandbox" blbe i u sql i u docker transformaci
